src/prometheus_alert_rules/blackbox_missing.rule

```diff
@@ -0,0 +1,13 @@
+# Based on https://awesome-prometheus-alerts.grep.to/rules.html#prometheus-self-monitoring-1
+alert: BlackboxJobMissing
+expr: absent(up{})
```

Comment:
I forget, is this going to trigger on startup?

Reply:
I'm not sure; we have this for alertmanager, so if it does there, it'll also happen here